doc/introduction/data.rst

```diff
@@ -0,0 +1,112 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_data:
+
+Quantum Datasets
+================
+
+PennyLane provides the :mod:`~.data` subpackage to download, create, store and manipulate quantum datasets within the PennyLane framework.
+
+Loading Datasets in PennyLane
+-----------------------------
+
+We can access data of a desired type with the :func:`~pennylande.data.load` function. This downloads the desired
+dataset or loads it from local storage if previously downloaded. Some examples of data categories available for download include 'qchem',
```

Comment:
Should it be \``"qchem"\`` or something similar?

Reply:
Yes, exactly! Thanks Soran, I was looking for that formatting. 
This line is gone now, but I will apply this everywhere else that it appears.